prymer/offtarget/bwa.py

```diff
@@ -201,6 +201,7 @@ class BwaAlnInteractive(ExecutableRunner):
         reverse_complement: reverse complement each query sequence before alignment.
         include_alt_hits: if True include hits to references with names ending in _alt, otherwise
                           do not include them.
+        header: the SAM alignment header.
```

Comment:
This adds the header as a public attribute because that seems useful.

I can revert if we don't want to make this available!

Reply:
- [ ] I should add a unit test for this